docs/modules/user-guide/partials/proc_adding-volume-components.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc_adding-volume_{context}"]
+= Adding `volume` components to a devfile
+
+[role="_abstract"]
+Use volume components in order to share files among container components, making it easier to collaborate with other teams during the development process.
+
+.Prerequisites
+
+* xref:adding-schema-version-to-a-devfile.adoc[]
+* xref:adding-a-name-to-a-devfile.adoc[]
+* xref:adding-components-to-a-devfile.adoc[]
+
+.Procedure
+
+* Define a component using the type `volume`.
```

Comment:
Would this phrasing create a more direct association with the example that follows?
```suggestion
* Define a `volume` component in your devfile.
```